packages/common/src/permissions/_internal/checkEntityPolicy.ts

```diff
@@ -51,13 +50,26 @@ export function checkEntityPolicy(
 
   if (type === "group-admin") {
```

Comment:
maybe this should be `group-owner-or-admin` since you check for either?

Reply:
I think it depends what we're checking b/c from the platform perspective, an owner has all the powers of an admin, and a member, but they do not have separate entries as `member` and `admin`. 

So - if we want to allow "owner" users to access things gated to "at least" admin, then we want the inclusive approach. 

Locally I've added `is-strict-group-admin` and `is-strict-group-member`, which check for that strict value, vs the platform transitive model. Lets discuss over a call

Reply:
For now, we will stick with the platform semantics; if needed we can add the strict semantics.